src/url.js

```diff
@@ -367,7 +367,7 @@ export default function url(publicId, options = {}, config = {}) {
     throw error;
   }
   let resultUrl = urlString(publicId, options);
-  if(options.analytics) {
+  if(options.urlAnalytics) {
```

Comment:
AnalyticsOptionsParameters in the types should also be changed?

Reply:
yup updated